dist_run.py

```diff
@@ -199,7 +302,7 @@ def main():
 
     # Load weights
     logger.info(f"Loading weights for {pp_rank=} on {device=}")
-    with TrackTime("cuda") as timer:
+    with TrackTime() as timer:
```

Comment:
I believe this change means that the load itself is synchronous and hence not need for the cuda timer?

Reply:
that's my understanding but to be safe just reverted it back to CUDAtracktime.  We should be careful though as chat uses generic perf_timer atm for their load so if it's not synchronous, we'll not be comparing apples to apples.